db/mob_db2.conf

```diff
@@ -94,6 +94,15 @@ mob_db: (
 		// ...
 	}
 	DamageTakenRate: damage taken rate    (int, defaults to 100)
+	Inherit: true/false                   (bool, defaults to false.
+                                               When true, inherits the
+                                               values from mob_db using
+                                               Id as the primary key.
+                                               Individual items in a nested 
+                                               field (e.g., Drops) can be
```

Comment:
Is this bit about overriding drops documented elsewhere with this wording? Because I don't think it's exactly correct, I believe it's currently a bit complicated to properly override drops without breaking anything

If I'm not missing anything in the existing code, it's currently possible to override each drop slot sequentially, but there's no way to cleanly override a slot while leaving the previous ones untouched

Reply:
Thanks, fixed now.